### PR TITLE
Added possibility to use callable when setting 'default_stage'.

### PR DESCRIPTION
**Pull request by kszymukowicz**
| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | Yes
| BC breaks?    | No
| Deprecations? | No
| Fixed tickets | #1278

### CODE REVIEW

**Comment by kszymukowicz** [comment]
The docs:
https://github.com/deployphp/docs/pull/94

File: src/Deployer.php [hunk 1]
```diff
@@ -116,7 +116,11 @@ public function __construct(Application $console)
             return new Task\ScriptManager($c['tasks']);
         };
         $this['hostSelector'] = function ($c) {
-            return new Host\HostSelector($c['hosts'], $c['config']['default_stage']);
+            $default_stage = $c['config']['default_stage'];
```
Comment (antonmedv) [inline src/Deployer.php]: please rename `$default_stage` to `$defaultStage`.

**Comment by kszymukowicz** [comment]
Thank you @antonmedv !
What is the chance that you will merge it to 4.3 if I will make PR ?

**Comment by antonmedv** [comment]
I don't support 4.x anymore. But if you want I can merge it there and create new release.

**Comment by kszymukowicz** [comment]
That would be great!!! I will prepare a PR for 4.3 branch then :)

**Pull request merged**